SimpleList/src/main.js

```diff
@@ -41,6 +41,11 @@ LENS.addEventListener('refocus.lens.load', () => {
         } else if (chg['sample.update']) {
           samples.set(chg['sample.update'].new.name.toLowerCase(),
             chg['sample.update'].new);
+        } else if (chg['sample.nochange']) {
+          samples.set(chg['sample.nochange'].name.toLowerCase(),
+            chg['sample.nochange']);
```

Comment:
why set if it hasn't changed?

Reply:
Right - I should just be updating its "last updated" timestamp - setting is incorrect here, since the shape of the "sample" in the event data here is different

Reply:
@jgraff2 updated